__tests__/Validate.test.js

```diff
@@ -15,39 +9,39 @@ export const getLogSpy = () => {
 describe('자동차 이름 입력 검증 테스트', () => {
   test('자동차 이름 빈 값 오류 검증 테스트', () => {
     //then
-    expect(() => checkIsEmpty('')).toThrow('[ERROR]');
+    expect(() => Validate.checkIsEmpty('')).toThrow('[ERROR]');
   });
 
   test.each([['앵버앵버앵버', '상']])('자동차 이름 길이(1~5자) 오류 검증 테스트', (carName) => {
     //then
-    expect(() => checkCarNameLength(carName)).toThrow('[ERROR]');
+    expect(() => Validate.checkCarNameLength(carName)).toThrow('[ERROR]');
   });
 
   test('자동차 이름 중복 오류 검증 테스트', () => {
     // given
     const carNames = ['상추', '상추'];
```

Comment:
이런것도 변수화 하여 의미를 더 강조하면 어떨까요?

```suggestion
    const name = '상추';
    const carNames = [name, name];
```